Add_user.css

```diff
@@ -20,52 +27,62 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
 	overflow:auto;
 	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
-	float:left;
+	float:right;
 }
 
 
-
+/* Set up Drop down menu for */
 nav a{
 	display:block;
 	width:100px;
 	padding: 20px 15px;
 	text-decoration:none;
-	font-family:Arial;
-	color:black;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:bold;
+	color:white;
 	text-align: center;
 }
 
 nav a:hover{
-	background:purple;
+	background:black;
 	transition:0.5s;
 }
 
 ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 130px;
 	width: 100px;
 	display: none;
 	visibility: hidden;
 	z-index: 5;
+}
 
+ul li #a{
```

Comment:
I fit the error of  the drop down menu for the setting because if I don't the then when I hover on setting it still display a drop down menu that hold the right function But the column is display underneath the Music play and it mean I can see it but can't access to it and it. This will denial the admin to be able to access the setting which can have about influence and make my music database do become low-quality music database and that's not what I wanted